### DISCUSSION TITLE
Issues with keymaps

### DISCUSSION THREAD

**Discussion by WildKyuri** [LazyVim/LazyVim#4011] [General]
# Keymaps not working at all
Arrived home recently, updated LazyVim, and now I'm having issues with the keymaps, save and quit not working with "<space+w>" and "<space>q", which-key pop out but not doing what it has to do (I mean space = leader)

**Comment by rogeriofrsouza** [comment]
As commented here https://github.com/LazyVim/LazyVim/discussions/4008#discussioncomment-10035729 that's probably because `which-key` v3 has a new mappings format and you need to update your configuration.

There was a recent commit fixing some code with the old format https://github.com/LazyVim/LazyVim/commit/702471e454ff552189ab03325dc19d0f7a601704
And you can see more examples in the plugin's docs: https://github.com/folke/which-key.nvim?tab=readme-ov-file#%EF%B8%8F-setup

**Comment by folke** [comment]
If the problem persists after updating which-key.nvim, please submit an issue (in the wk repo)
